tuning/config/peft_config.py

```diff
@@ -14,16 +14,38 @@
 
 # Standard
 from dataclasses import dataclass, field
+from enum import Enum
 from typing import List
 
+# Third Party
+from transformers import Mxfp4Config as HfMxfp4Config
+
+
+class QUANT_METHOD(Enum):
+    MXFP4 = "mxfp4"
+
+
+class PEFT_METHOD(Enum):
+    PT = "pt"
+    LORA = "lora"
+    ALORA = "alora"
+
+
+@dataclass
+class Mxfp4Config:
+    dequantize: bool = True
+
+    def to_hf_config(self):
+        return HfMxfp4Config(dequantize=self.dequantize)
+
 
 @dataclass
 class LoraConfig:
     """
     This is the configuration class to store the configuration of a [`LoraModel`].
 
     Args:
-        r (`int`):
+        lora_r (`int`):
```

Comment:
@ashokponkumar would you welcome this change along with this PR? if not i can revert it